pkg/testing/v1alpha1/service.go

```diff
@@ -63,10 +62,29 @@ func ServiceWithoutNamespace(name string, so ...ServiceOption) *v1alpha1.Service
 	for _, opt := range so {
 		opt(s)
 	}
+	return s
+}
+
+// DefaultService creates a service with ServiceOptions and with default values set
+func DefaultService(name, namespace string, so ...ServiceOption) *v1alpha1.Service {
+	s := &v1alpha1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+	}
+	for _, opt := range so {
+		opt(s)
+	}
 	s.SetDefaults(context.Background())
 	return s
```

Comment:
Why not reuse the existing Service function by calling it?

Service(name, namespace, so, WithServiceDefaults)